### DISCUSSION TITLE
What would of the Tic-80 machine looked like if it was physical?

### DISCUSSION THREAD

**Discussion by spooneystone** [nesbox/TIC-80#1883] [General]
I imagine there is a few users of tic-80 who may be skilled in 3d modeling, and i was thinking.... what would the tic-80 look like in its design?
was thinking it might be a good idea to get some 3d models on show, and who knows.. maybe some people might make some cool cases for the raspberry pi running tic-80

**Comment by scratchminer** [comment]
I'm not really a big user of TIC-80 or skilled in 3D modeling, but I made these because I wanted to:
<img width="922" alt="Screenshot 2023-01-22 at 5 16 12 PM" src="https://user-images.githubusercontent.com/121195139/213943257-0ab536db-9b9e-4216-a058-e8a5898d71f2.png">
<img width="922" alt="Screenshot 2023-01-22 at 5 16 23 PM" src="https://user-images.githubusercontent.com/121195139/213943258-f03626be-a957-4d87-aa56-0e1b36d6a89c.png">

**Reply by spooneystone** [reply]
Definitely a blue plastic would be the choice. Maybe with white keys and yellow accents on them.

**Comment by spooneystone** [comment]
Just letting people know there is a hardware section on the Tic-80 Discord.. https://discord.gg/9MjF7s6g
